control-plane-operator/controllers/hostedcontrolplane/configoperator/reconcile.go

```diff
@@ -219,6 +219,7 @@ var (
 	volumeMounts = util.PodVolumeMounts{
 		hccContainerMain().Name: util.ContainerVolumeMounts{
 			hccVolumeKubeconfig().Name:      "/etc/kubernetes/kubeconfig",
+			hccVolumeInfraKubeconfig().Name: "/etc/kubernetes/infra-kubeconfig",
```

Comment:
I think we should add the term`kubevirt` somewhere here in the path. `/etc/kubernetes/kubevirt-infra-kubeconfig`